fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -206,6 +206,7 @@ DELETE: 'DELETE';
 DEMAND: 'DEMAND';
 DESC: 'DESC';
 DESCRIBE: 'DESCRIBE';
+DETAIL: 'DETAIL';
```

Comment:
what's the 'DETAIL' keyword for, the show data command use 'ALL' keyword not 'DETAIL'?

Reply:
I have deleted it.